common.gypi

```diff
@@ -469,14 +454,8 @@
             'ldflags!': [ '-pthread' ],
           }],
           [ 'OS=="aix"', {
-            'variables': {'real_os_name': '<!(uname -s)',},
+            'cflags': [ '-maix64' ],
```

Comment:
This is also on L460 below in the `'target_arch=="ppc64"'` block -- Is this also needed on IBM i (in which case keep this line and drop the one in the conditional)?

Reply:
A real game of whack-a-mole.
Maybe I should remove all conditions and add line until the build stops failing 😉 
(I actually would, but we don't have `IBMi` in our CI cluster)

Reply:
Merged